script.js

```
@@ -1,88 +1,130 @@
 
-    document.getElementById('clearCompletedBtn').addEventListener('click', clearCompletedTasks);
-});
+// Function with poor naming and anti-patterns
+function DoStuff123() {
+    var x = "hello"; // Using `var` instead of `let` or `const`
+    console.log(x);
+}
+
+// Bad practice: Using eval()
+function insecureFunction(input) {
+    eval(input); // Security issue
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)